src/app/tables/components/TableSortingSelecting.tsx

```diff
@@ -252,11 +249,91 @@ function EnhancedTableToolbar(props: EnhancedTableToolbarProps) {
           </IconButton>
         </Tooltip>
       ) : (
-        <Tooltip title="Filter list">
-          <IconButton>
-            <FilterListIcon />
-          </IconButton>
-        </Tooltip>
+        <Grid container spacing={2} alignItems="center">
+          {/* Grupo de botones "Nuevo" */}
```

Comment:
It is a good practice to use always English in the code, even for comments.

Reply:
I have already changed it to English